docker/README.md

```diff
@@ -0,0 +1,13 @@
+# Containers
```

Comment:
Thank you for including a README with some instructions!

Reply:
And I was felling bad that the README was way too thin for my tastes :)